test/functional/FunctionalCodeAction.hs

```diff
@@ -353,6 +355,12 @@ unusedTermTests = testGroup "unused term code actions" [
               $ Just CodeActionQuickFix `notElem` kinds
     ]
 
+expectFailIfGhc92 :: String -> TestTree -> TestTree
+expectFailIfGhc92 reason =
```

Comment:
please, use the existing `knownBrokenForGhcVersions` like
https://github.com/haskell/haskell-language-server/blob/5714207bfe44b13b02726334f9caf883249e68a9/plugins/hls-hlint-plugin/test/Main.hs#L347

Reply:
from https://github.com/haskell/haskell-language-server/blob/master/hls-test-utils/src/Test/Hls/Util.hs#L32